packages/loopring_v3/contracts/amm/libamm/AmmJoinProcess.sol

```diff
@@ -41,6 +41,7 @@ library AmmJoinProcess
     {
         require(!ctx.settings.joinsDisabled, "JOINS_DISABLED");
         require(join.validUntil >= block.timestamp, "EXPIRED");
+        // require(!S.exitMode, "JOIN_DISALLOWED_IN_EXIT_MODE")
```

Comment:
I think skipping the check should be fine?

Reply:
Yes I think we can skip it. I guess in most cases it's not smart to join a pool in exit mode but it should be valid.

Reply:
OK

Reply:
I'll leave the code commented out then.